Formula/git-big-picture.rb

```diff
@@ -0,0 +1,26 @@
+class GitBigPicture < Formula
+  include Language::Python::Virtualenv
+  desc "Visualization tool for Git repositories"
+  homepage "https://github.com/git-big-picture/git-big-picture"
+  url "https://github.com/git-big-picture/git-big-picture/archive/refs/tags/v1.2.2.tar.gz"
+  sha256 "7b2826d72e146c7a53e7a1cc9533c360cd8e0feb870c7d1eadcc189b8bc2c5f6"
+  license "GPL-3.0-or-later"
+
+  depends_on "graphviz"
+  depends_on "python@3.11"
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    cd testpath do
+      system "git", "init"
+      system "git", "config", "user.name", "Dummy author"
+      system "git", "config", "user.email", "dummy@domain.test"
+      system "git", "commit", "--allow-empty", "-m", "Empty commit"
+      system "git", "big-picture", "-f", "svg", "-o", "output.svg"
+      assert_predicate Pathname.pwd/"output.svg", :exist?
+    end
```

Comment:
`testpath` is already the working directory.
```suggestion
    system "git", "init"
    system "git", "config", "user.name", "Dummy author"
    system "git", "config", "user.email", "dummy@domain.test"
    system "git", "commit", "--allow-empty", "-m", "Empty commit"
    system "git", "big-picture", "-f", "svg", "-o", "output.svg"
    assert_path_exists testpath/"output.svg"
```
I'd try skipping the `git config` steps too, since I think `brew` sets that up automatically for the test.

Reply:
@carlocab done x2 :+1: